Basic/Basic/MainViewController.m

```diff
@@ -24,6 +24,8 @@ @implementation MainViewController
 
 - (void) viewDidLoad
 {
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(changeOrder:) name:@"ChangeOrderNotification" object:nil];
```

Comment:
If you add an observer in NSNotificationCenter you must always remove this observer at some point.